Fixes this configure error:

Node.js configure: Found Python 3.9.2...
Traceback (most recent call last):
  File "/opt/local/var/macports/build/devel_nodejs15/nodejs15/work/node-v15.13.0/./configure", line 26, in <module>
    import configure
  File "/opt/local/var/macports/build/devel_nodejs15/nodejs15/work/node-v15.13.0/configure.py", line 1888, in <module>
    configure_openssl(output)
  File "/opt/local/var/macports/build/devel_nodejs15/nodejs15/work/node-v15.13.0/configure.py", line 1455, in configure_openssl
    variables['openssl_quic'] = b(getsharedopensslhasquic.get_has_quic(options.__dict__['shared_openssl_includes']))
  File "tools/getsharedopensslhasquic.py", line 12, in get_has_quic
    f = open(openssl_crypto_h)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/local/include/openssl/openssl/crypto.h'


--- tools/getsharedopensslhasquic.py.orig	2021-03-30 21:20:12.000000000 -0700
+++ tools/getsharedopensslhasquic.py	2021-03-31 12:35:14.000000000 -0700
@@ -6,7 +6,6 @@
   if include_path:
     openssl_crypto_h = os.path.join(
         include_path,
-        'openssl',
         'crypto.h')
 
     f = open(openssl_crypto_h)
